src/libraries/System.Threading.Tasks.Dataflow/ref/System.Threading.Tasks.Dataflow.csproj

```diff
@@ -1,11 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.1;netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.1;netstandard2.0;net461</TargetFrameworks>
     <Nullable>enable</Nullable>
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="System.Threading.Tasks.Dataflow.cs" />
     <Compile Include="System.Threading.Tasks.Dataflow.netstandard21.cs"
-             Condition="'$(TargetFramework)' == 'netstandard2.1'" />
+             Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'netstandard2.1'))" />
```

Comment:
I assume this means the `.netstandard21.cs` file will be included on `NetCoreAppCurrent`.

Reply:
Correct. On all target frameworks which can reference netstandard2.1 assemblies: netstandard2.1 and >= netcoreapp2.1.